bot/src/domain/activity/MapActivity.ts

```
@@ -109,17 +109,15 @@ export class MapActivity {
     switch (format) {
-      case "hour":
-        return `${date.getFullYear()}-${
-          date.getMonth() + 1
-        }-${date.getDate()}-${date.getHours()}`;
-      case "day":
+      case 'hour':
+        return `${date.getFullYear()}-${date.getMonth() + 1}-${date.getDate()}-${date.getHours()}`;
+      case 'day':
         return `${date.getFullYear()}-${date.getMonth() + 1}-${date.getDate()}`;
-      case "raw":
+      case 'raw':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)